integration/js/README.md

```diff
@@ -33,7 +34,10 @@ chmod +x configureMac.sh
 ./configureMac.sh
 ```
 
-Choose 'y'.
+Choose 'y'. <br />
```

Comment:
is the `<br />`  necessary?

Reply:
It should not be :)

Reply:
I just did it mainly for formatting. I want to make sure Note is on next line because Choose 'y' is part of the script above. Please let me know if you want me to remove that br. 

I can also use two spaces to add the line break if you prefer that. I chose `<br />` as that is easier to read.

Reply:
Let's use a line break since that is what the rest of the document does :).

Reply:
Done, thanks.